kre-go/main.go

```diff
@@ -1,34 +1,49 @@
 package kre
 
 import (
+	"fmt"
 	"os"
 	"os/signal"
+	"strings"
 	"syscall"
 	"time"
 
+	"github.com/konstellation-io/kre/libs/simplelogger"
 	"github.com/nats-io/nats.go"
-	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/types/known/anypb"
 
 	"github.com/konstellation-io/kre-runners/kre-go/config"
 	"github.com/konstellation-io/kre-runners/kre-go/mongodb"
-	"github.com/konstellation-io/kre/libs/simplelogger"
 )
 
 // HandlerInit is executed once. It is useful to initialize variables that will be constants
 // between handler calls.
 type HandlerInit func(ctx *HandlerContext)
 
-// Handler is the function executed each time a message from NATS arrives. This function must return
-// the protobuf for the next node or the final response if it is the last one.
-type Handler func(ctx *HandlerContext, data *anypb.Any) (proto.Message, error)
+// Handler is the function executed each time a message from NATS arrives.
+//
+// Responses, if desired, must be sent through the handlerContext's sendOutput func.
```

Comment:
If I'm not mistaken, there are two ways of sending responses between nodes with `sendOutput` and `sendAny` functions.

```suggestion
// Responses, if desired, must be sent through the handlerContext's sendOutput or SendAny funcs.
```